consensus/src/counters.rs

```diff
@@ -78,166 +58,167 @@ pub static ref EPOCH_CHANGE_DEQUEUED_MSGS: IntCounter = OP_COUNTERS.counter("epo
-pub static ref CREATION_TO_RECEIVAL_S: DurationHistogram = OP_COUNTERS.duration_histogram("creation_to_receival_s");
+pub static ref CREATION_TO_RECEIVAL_S: DurationHistogram = DurationHistogram::new(register_histogram!("libra_consensus_creation_to_receival_s", "Duration between block generation time until the moment it is received and ready for execution.").unwrap());
 
 ////////////////////////////////////
 // PROPSOSAL/VOTE TIMESTAMP COUNTERS
 ////////////////////////////////////
 /// Count of the proposals that passed the timestamp rules and did not have to wait
-pub static ref PROPOSAL_NO_WAIT_REQUIRED_COUNT: IntCounter = OP_COUNTERS.counter("proposal_no_wait_required_count");
+pub static ref PROPOSAL_NO_WAIT_REQUIRED_COUNT: IntCounter = register_int_counter!("libra_consensus_proposal_no_wait_required_count", "Count of the proposals that passed the timestamp rules and did not have to wait").unwrap();
 
 /// Count of the proposals where passing the timestamp rules required waiting
-pub static ref PROPOSAL_WAIT_WAS_REQUIRED_COUNT: IntCounter = OP_COUNTERS.counter("proposal_wait_was_required_count");
+pub static ref PROPOSAL_WAIT_WAS_REQUIRED_COUNT: IntCounter = register_int_counter!("libra_consensus_proposal_wait_was_required_count", "Count of the proposals where passing the timestamp rules required waiting").unwrap();
 
 /// Count of the proposals that were not made due to the waiting period exceeding the maximum allowed duration, breaking timestamp rules
-pub static ref PROPOSAL_MAX_WAIT_EXCEEDED_COUNT: IntCounter = OP_COUNTERS.counter("proposal_max_wait_exceeded_count");
+pub static ref PROPOSAL_MAX_WAIT_EXCEEDED_COUNT: IntCounter = register_int_counter!("libra_consensus_proposal_max_wait_exceeded_count", "Count of the proposals that were not made due to the waiting period exceeding the maximum allowed duration, breaking timestamp rules").unwrap();
 
 /// Count of the proposals that were not made due to waiting to ensure the current time exceeds min_duration_since_epoch failed, breaking timestamp rules
-pub static ref PROPOSAL_WAIT_FAILED_COUNT: IntCounter = OP_COUNTERS.counter("proposal_wait_failed_count");
+pub static ref PROPOSAL_WAIT_FAILED_COUNT: IntCounter = register_int_counter!("libra_consensus_proposal_wait_failed_count", "Count of the proposals that were not made due to waiting to ensure the current time exceeds min_duration_since_epoch failed, breaking timestamp rules").unwrap();
 
 /// Histogram of time waited for successfully proposing a proposal (both those that waited and didn't wait) after following timestamp rules
-pub static ref PROPOSAL_SUCCESS_WAIT_S: DurationHistogram = OP_COUNTERS.duration_histogram("proposal_success_wait_s");
+pub static ref PROPOSAL_SUCCESS_WAIT_S: DurationHistogram = DurationHistogram::new(register_histogram!("libra_consensus_proposal_success_wait_s", "Histogram of time waited for successfully proposing a proposal (both those that waited and didn't wait) after following timestamp rules").unwrap());
```

Comment:
same here, state = success / failure

Reply:
Skipping this for now as DurationHistogram is a wrapper over Histogram and it it not as trivial to refactor this metric.